src/ServicePulse.Host/vue/src/components/failedmessages/AllFailedMessages.vue

```diff
@@ -141,18 +141,18 @@ function exportSelected() {
   }
 
   function parseObject(obj, propertiesToSkip, path) {
-    if (path == undefined) path = "";
+    if (path === undefined) path = "";
```

Comment:
should be comparing to == null rather than === undefined I would think?

Reply:
We are using the ["smart" mode](https://eslint.org/docs/latest/rules/eqeqeq#smart), and it seems to suggest `===`.
@PhilBastian is the "smart" mode the wrong setting?